src/webgpu/api/validation/queue/copyToTexture/ImageBitmap.spec.ts

```diff
@@ -329,19 +317,13 @@ g.test('destination_texture,format')
     const dstTexture = t.device.createTexture({
       size: { width: 1, height: 1, depthOrArrayLayers: 1 },
       format,
-      usage: GPUTextureUsage.COPY_DST,
+      usage: GPUTextureUsage.COPY_DST | GPUTextureUsage.RENDER_ATTACHMENT,
     });
     t.device.popErrorScope();
 
-    const success = kValidTextureFormatsForCopyIB2T.includes(format);
+    const success = ([...kValidTextureFormatsForCopyIB2T] as string[]).includes(format);
```

Comment:
This introduces unnecessary runtime cost, can it simply be:

```suggestion
    const success = (kValidTextureFormatsForCopyIB2T as readonly string[]).includes(format);
```